ksql-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java

```diff
@@ -35,7 +35,9 @@
 
   private final PreparedStatement<T> statement;
   @EffectivelyImmutable
-  private final ImmutableMap<String, Object> overrides;
+  private final ImmutableMap<String, Object> clientProperties;
```

Comment:
by `clientProperties`, do you actually mean requestProperties?  may be we can stick to that?

Reply:
renamed to `RequestProperties`